orchestration/templates/stage-data.yaml

```diff
@@ -83,7 +83,7 @@ spec:
           {{- with .Values.dataflow }}
           - --project={{ .project }}
           - --region={{ .region }}
-          - --jobName=hca-stage-metadata-{{ $stagingPrefix }}
+          - --jobName=hca-stage-metadata-{{ regexReplaceAll "[^-a-z0-9]+" $stagingPrefix "-" }}
```

Comment:
Where does regexReplaceAll come from?

Reply:
revisiting this, it comes from helm, which i mistakenly thought was accessible from argo

Reply:
update: helm is accessible from argo. the syntax here was incorrect for function invocations, however. those need to be wrapped in hyphens in addition to double braces. updated.